src/doom/p_inter.c

```diff
@@ -544,57 +544,57 @@ P_TouchSpecialThing
 	
 	// ammo
       case SPR_CLIP:
-	if (special->flags & MF_DROPPED)
-	{
-	    if (!P_GiveAmmo (player,am_clip,0))
-		return;
-	}
-	else
-	{
-	    if (!P_GiveAmmo (player,am_clip,1))
-		return;
-	}
+	if (!P_GiveAmmo (player,am_clip, 1,
```

Comment:
Please comment out this block with `/* ... */` instead of removing it.

Reply:
Again, back to using this for the time being.